bundle-analyzer is in dev-dependencies but needed in production -> crash at startup

### Issue Summary

bundle-analyzer is declared in dev-dependencies but needed in production -> crash at startup

### Steps to Reproduce

1. Deploy the commit 86d292838c05dddb0205b97163c0fc78c2b72ebc 
2. Remove dev dependencies (automatly done by PaaS solution)
3. The don't start, with this error 

```
2021-10-19 09:39:23.129453655 +0000 UTC [web-1] ready - started server on 0.0.0.0:23450, url: http://localhost:23450
2021-10-19 09:39:23.586138399 +0000 UTC [web-1] Error: failed to load next.config.js, see more info here https://nextjs.org/docs/messages/next-config-error
2021-10-19 09:39:23.588127698 +0000 UTC [web-1] Error: Cannot find module '@next/bundle-analyzer'
2021-10-19 09:39:23.588129394 +0000 UTC [web-1] Require stack:
2021-10-19 09:39:23.588135298 +0000 UTC [web-1] - /app/next.config.js
2021-10-19 09:39:23.588137254 +0000 UTC [web-1] - /app/node_modules/next/dist/server/config.js
2021-10-19 09:39:23.588156921 +0000 UTC [web-1] - /app/node_modules/next/dist/server/next.js
2021-10-19 09:39:23.588157473 +0000 UTC [web-1] - /app/node_modules/next/dist/server/lib/start-server.js
2021-10-19 09:39:23.588182277 +0000 UTC [web-1] - /app/node_modules/next/dist/cli/next-start.js
2021-10-19 09:39:23.588182807 +0000 UTC [web-1] - /app/node_modules/next/dist/bin/next
2021-10-19 09:39:23.588222542 +0000 UTC [web-1] at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
2021-10-19 09:39:23.588223118 +0000 UTC [web-1] at Function.mod._resolveFilename (/app/node_modules/next/dist/build/webpack/require-hook.js:96:28)
2021-10-19 09:39:23.588239667 +0000 UTC [web-1] at Function.Module._load (internal/modules/cjs/loader.js:746:27)
2021-10-19 09:39:23.588258099 +0000 UTC [web-1] at Module.require (internal/modules/cjs/loader.js:974:19)
2021-10-19 09:39:23.588258922 +0000 UTC [web-1] at require (internal/modules/cjs/helpers.js:93:18)
2021-10-19 09:39:23.588386457 +0000 UTC [web-1] at Object.<anonymous> (/app/next.config.js:5:28)
2021-10-19 09:39:23.588388630 +0000 UTC [web-1] at Module._compile (internal/modules/cjs/loader.js:1085:14)
2021-10-19 09:39:23.588389151 +0000 UTC [web-1] at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
2021-10-19 09:39:23.588389828 +0000 UTC [web-1] at Module.load (internal/modules/cjs/loader.js:950:32)
2021-10-19 09:39:23.588390269 +0000 UTC [web-1] at Function.Module._load (internal/modules/cjs/loader.js:790:12) {
2021-10-19 09:39:23.588390694 +0000 UTC [web-1] code: 'MODULE_NOT_FOUND',
2021-10-19 09:39:23.588391518 +0000 UTC [web-1] requireStack: [
2021-10-19 09:39:23.588391931 +0000 UTC [web-1] '/app/next.config.js',
2021-10-19 09:39:23.588414760 +0000 UTC [web-1] '/app/node_modules/next/dist/server/config.js',
2021-10-19 09:39:23.588418692 +0000 UTC [web-1] '/app/node_modules/next/dist/server/next.js',
2021-10-19 09:39:23.588419542 +0000 UTC [web-1] '/app/node_modules/next/dist/server/lib/start-server.js',
2021-10-19 09:39:23.588419958 +0000 UTC [web-1] '/app/node_modules/next/dist/cli/next-start.js',
2021-10-19 09:39:23.588433850 +0000 UTC [web-1] '/app/node_modules/next/dist/bin/next'
2021-10-19 09:39:23.588436701 +0000 UTC [web-1] ]
2021-10-19 09:39:23.588437595 +0000 UTC [web-1] }
```

--- a/package.json
+++ b/package.json
@@ -37,6 +37,7 @@
     "@hookform/resolvers": "^2.8.1",
     "@jitsu/sdk-js": "^2.2.4",
     "@prisma/client": "^2.30.2",
+    "@next/bundle-analyzer": "11.1.2",
     "@radix-ui/react-avatar": "^0.1.0",
     "@radix-ui/react-collapsible": "^0.1.0",
     "@radix-ui/react-dialog": "^0.1.0",
@@ -97,7 +98,6 @@
     "zod": "^3.8.2"
   },
   "devDependencies": {
-    "@next/bundle-analyzer": "11.1.2",
     "@trivago/prettier-plugin-sort-imports": "2.0.4",
     "@types/async": "^3.2.7",
     "@types/bcryptjs": "^2.4.2",

